selfdrive/car/hyundai/values.py

```diff
@@ -449,6 +452,17 @@ def match_fw_to_car_fuzzy(live_fw_versions, offline_fw_versions) -> Set[str]:
       obd_multiplexing=False,
     ),
 
+    # CAN & CAN FD query to understand the three digit date code
+    # HDA2 cars usually use 6 digit date codes, so skip bus 1
+    Request(
+      [HYUNDAI_ECU_MANUFACTURING_DATE],
+      [HYUNDAI_VERSION_RESPONSE],
```

Comment:
This may be the calibration/programming date, and the date in the full FW version string may be the manufacture date, as on the EV6, this date was 2 months after the date in the string

Reply:
`b'\xf1\x00CV1 MFC  AT USA LHD 1.00 1.05 99210-CV000 211027'`

vs

`b'\xf1\x8b !\x12 '`